app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1623,8 +1623,8 @@ private void passInfoToSheet(final Place place) {
             if (fabGallery.isShown()) {
                 Timber.d("Gallery button tapped. Place: %s", selectedPlace.toString());
                 storeSharedPrefs(selectedPlace);
-
-                controller.initiateGalleryPick(getActivity(), false);
+                
```

Comment:
Can you revert this change? :)

Reply:
So.. Should i add If-else condition like I had before? @madhurgupta10 ? 😀

Reply:
Sorry for not being clear, can you remove the extra empty line?